samples/detect.js

```diff
@@ -23,24 +23,40 @@ const {Transform, pipeline} = require('stream');
 const pump = util.promisify(pipeline);
 
 function detectTextIntent(projectId, sessionId, queries, languageCode) {
+  const PROJECT_ID = projectId;
+  const SESSION_ID = sessionId;
+  const QUERIES = queries;
+  const LANGUAGE_CODE = languageCode;
   // [START dialogflow_detect_intent_text]
+
+  /**
+   * TODO(developer): UPDATE these variables before running the sample.
+   */
+  // const PROJECT_ID = 'PROJECT_ID';
+  // const SESSION_ID = String(Date.now()); // sessionId can be a random number or some type of user identifier (preferably hashed)
+  // const QUERIES = [
```

Comment:
Can we rename `QUERIES` to something like `QUERY_TEXTS` just to try to match the naming used by the API. 

The object is QueryInput, but the field is still `text`. 

https://cloud.google.com/dialogflow/docs/reference/rpc/google.cloud.dialogflow.v2#google.cloud.dialogflow.v2.QueryInput

Thoughts?

Reply:
Makes sense.